extension/src/experiments/data/index.ts

```diff
@@ -86,9 +86,12 @@ export class ExperimentsData extends BaseData<ExperimentsOutput> {
 
     return createFileSystemWatcher(
       disposable => this.dispose.track(disposable),
-      getRelativePattern(dotGitPath, '**'),
```

Comment:
I couldn't figure out why but this pattern (`.git/**`) wouldn't work correctly, only firing for files directly in the .git folder ([more details in slack thread](https://iterativeai.slack.com/archives/C01RB7BTG4C/p1675103470404509)). The only solution I could come up with was having our `.git` watcher search the entire workspace folder and return if the fired path wasn't in the `.git` folder.

Reply:
We can't take this approach as the folder can actually be outside of the workspace and/or the dvc root (e.g monorepo).

Reply:
Oh, you're correct, I didn't think about that! Will go ahead and close this pr.